team-manager/src/test/unit/java/ua/knu/knudev/teammanager/service/DepartmentServiceTest.java

```diff
@@ -23,9 +28,10 @@
 @ExtendWith(MockitoExtension.class)
 class DepartmentServiceTest {
 
-    private static final Department TEST_DEPARTMENT = getTestDepartment();
     private static final Double TEST_SPECIALTY_ID = DepartmentTestsConstants.TEST_SPECIALTY_ID;
     private static final UUID TEST_DEPARTMENT_ID = DepartmentTestsConstants.TEST_DEPARTMENT_ID;
+    private final UUID testDepartmentID = UUID.randomUUID();
+    private static final String TEST_DEPARTMENT_NAME = "Test Department";
```

Comment:
Put this to `DepartmentTestsConstants` if it doesnt exist already

Reply:
delete